Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -184,6 +226,49 @@ def save_failed_asset_joblogs(workspace: ly_test_tools._internal.managers.worksp
                     logger.warning(f"Error when saving log at path: {full_log_path}\n{e}")
 
 
+def split_batched_editor_log_file(workspace: ly_test_tools._internal.managers.workspace.AbstractWorkspaceManager,
+                                  starting_path: str,
+                                  destination_file: str) -> None:
+    """
+    Splits a batched editor log file into separate log files for each test case in the log
+    :param workspace: The LyTestTools Workspace object
+    :param starting_path: the original path for the logs
+    :param destination_file: the destination path for the logs
+    :return: None
+    """
+    if not destination_file:
+        logger.warning(f'No destination_file path found, got {destination_file} instead.')
+        raise FileNotFoundError
+    # text that designates the start of logging for a new test
+    test_case_split = ".py (testcase )"
+    dir_name = os.path.dirname(starting_path)
+
+    # the current log we are writing to
+    current_new_log_path = os.path.join(dir_name, f"SetUp" + ".log")
+    current_new_log = open(current_new_log_path, "a+")
+
+    # loop through the log to split, and write to the split logs
+    with open(destination_file) as log_file:
+        for line in log_file:
+            split_line = line.split(test_case_split)
+            if len(split_line) > 1:
+                # found a new test case, we need to split the log, line below is an example of what we're splitting
+                # <13:22:34> (python) - Running automated test: C:\\Git\\o3de\\AutomatedTesting\\Gem\\PythonTests\\
```

Comment:
see my comment above to avoid doing the parsing

Reply:
can you provide some sample code? it doesn't look trivial to me and if it isn't I'd rather cut a new ticket to fix that (especially since this PR didn't add this code, it simply copied it over from development).

Reply:
I will cut an internal ticket to get this updated.